lib/telly.rb

```diff
@@ -263,6 +263,8 @@ def Telly.testcase_id_from_beaker_script(beaker_file)
     # Find first matching line
     match = File.readlines(beaker_file).map { |line| line.match(TESTCASE_ID_REGEX) }.compact.first
 
+    raise Exception, 'Testcase ID could not be found in file' if match.nil?
```

Comment:
Is there a better, more specific error we can raise besides a generic `Exception`? Maybe `ArgumentError`?